src/tour.ts

```diff
@@ -16,6 +16,15 @@ export const tour = new Shepherd.Tour({
 const next = tour.next.bind(tour);
 const cancel = tour.cancel.bind(tour);
 
+// Dismiss tour on future runs once it is cancelled or completed.
+function dismissTour() {
+  if (!localStorage.getItem('shepherd-tour')) {
+    localStorage.setItem('shepherd-tour', 'yes');
+  }
+}
+tour.on('cancel', dismissTour);
+tour.on('complete', dismissTour);
```

Comment:
There is useful information in this tour, we should not automatically make it forever inaccessible to the user...  Can we change "cancel" to "dismiss" and add a checkbox for "don't show this on startup", and then create a "question mark" button in the top row of the application to re-show the dialog?